plugins/orchestrator-backend/package.json

```diff
@@ -60,6 +60,7 @@
     "@backstage/backend-app-api": "^0.7.2",
     "@backstage/backend-common": "^0.21.7",
     "@backstage/backend-dynamic-feature-service": "^0.2.9",
+    "@backstage/errors": "^1.2.4",
```

Comment:
Please run `yarn export-dynamic` on `plugins/orchestrator-backend` folder so this dependency is added to `dist-dynamic/package.json` file.

Reply:
ahh easy to miss out. thanks for the heads up.
any reasonable way to automatically enforce/warn against this?